core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -621,14 +621,22 @@ object SparkSubmit {
     if (isKubernetesCluster) {
       childMainClass = "org.apache.spark.deploy.kubernetes.submit.Client"
       if (args.isPython) {
+        childArgs += "--py-file"
```

Comment:
naming convention, for the sake of consistency, should make this `--primary-py-file`